Sources/Sentry/SentryStacktraceBuilder.m

```diff
@@ -33,15 +34,20 @@ - (SentryStacktrace *)retrieveStacktraceFromCursor:(SentryCrashStackCursor)stack
     NSMutableArray<SentryFrame *> *frames = [NSMutableArray array];
     SentryFrame *frame = nil;
     while (stackCursor.advanceCursor(&stackCursor)) {
+        SENTRY_LOG_DEBUG(@"Next stack frame: %s; address: %lu (SentryCrashSC_ASYNC_MARKER: %lu)",
+            stackCursor.stackEntry.symbolName, stackCursor.stackEntry.address,
+            SentryCrashSC_ASYNC_MARKER);
         if (stackCursor.stackEntry.address == SentryCrashSC_ASYNC_MARKER) {
             if (frame != nil) {
                 frame.stackStart = @(YES);
             }
             // skip the marker frame
+            SENTRY_LOG_DEBUG(@"Skipping async continuation marker frame.");
             continue;
         }
         if (stackCursor.symbolicate(&stackCursor)) {
-            [frames addObject:[self.crashStackEntryMapper mapStackEntryWithCursor:stackCursor]];
+            frame = [self.crashStackEntryMapper mapStackEntryWithCursor:stackCursor];
```

Comment:
Fixes a bug where it never detected async start frames.

Reply:
Can we put this in an extra PR, please?

Reply:
It was already in #2898, just need to merge main into this branch[